[BUG] Error CS0103: The name 'page' does not exist in the current context using sample code in documentation

### System info
- Playwright Version: 1.35.0
- Operating System: Windows Serve 2019, .net6.0
- Browser: Chromium

### Source code

The code is from the [documentation](https://playwright.dev/dotnet/docs/writing-tests).

```
using System.Text.RegularExpressions;
using System.Threading.Tasks;
using Microsoft.Playwright;
using Microsoft.Playwright.NUnit;
using NUnit.Framework;

namespace PlaywrightTests;

[Parallelizable(ParallelScope.Self)]
[TestFixture]
public class Tests : PageTest
{
    [Test]
    public async Task HomepageHasPlaywrightInTitleAndGetStartedLinkLinkingtoTheIntroPage()
    {
        await Page.GotoAsync("https://playwright.dev");

        // Expect a title "to contain" a substring.
        await Expect(Page).ToHaveTitleAsync(new Regex("Playwright"));

        // create a locator
        var getStarted = Page.GetByRole(AriaRole.Link, new() { Name = "Get started" });

        // Expect an attribute "to be strictly equal" to the value.
        await Expect(getStarted).ToHaveAttributeAsync("href", "/docs/intro");

        // Click the get started link.
        await getStarted.ClickAsync();

        // Expects page to have a heading with the name of Installation.
        await Expect(page
            .GetByRole(AriaRole.Heading, new() { Name = "Installation" }))
            .ToBeVisibleAsync();
    }
}
```

**Steps**
- dotnet test --settings PlaywrightTests.runsettings

**Expected**

Chromium browser to launch and test to run for https://playwright.dev

**Actual**

The following error is reported:

error CS0103: The name 'page' does not exist in the current context

--- a/docs/src/api/class-pageassertions.md
+++ b/docs/src/api/class-pageassertions.md
@@ -85,7 +85,7 @@ assertThat(page).not().hasURL("error");
 ```
 
 ```csharp
-await Expect(page).Not.ToHaveURL("error");
+await Expect(Page).Not.ToHaveURL("error");
 ```
 
 ## async method: PageAssertions.NotToHaveTitle
@@ -261,7 +261,7 @@ expect(page).to_have_title(re.compile(r".*checkout"))
 ```
 
 ```csharp
-await Expect(page).ToHaveTitle("Playwright");
+await Expect(Page).ToHaveTitle("Playwright");
 ```
 
 ### param: PageAssertions.toHaveTitle.titleOrRegExp
@@ -310,7 +310,7 @@ expect(page).to_have_url(re.compile(".*checkout"))
 ```
 
 ```csharp
-await Expect(page).ToHaveURL(new Regex(".*checkout"));
+await Expect(Page).ToHaveURL(new Regex(".*checkout"));
 ```
 
 ### param: PageAssertions.toHaveURL.urlOrRegExp

--- a/docs/src/api/class-playwrightassertions.md
+++ b/docs/src/api/class-playwrightassertions.md
@@ -157,7 +157,7 @@ PlaywrightAssertions.assertThat(page).hasTitle("News");
 ```
 
 ```csharp
-await Expect(page).ToHaveTitleAsync("News");
+await Expect(Page).ToHaveTitleAsync("News");
 ```
 
 ### param: PlaywrightAssertions.expectPage.page

--- a/docs/src/api/params.md
+++ b/docs/src/api/params.md
@@ -1589,7 +1589,7 @@ page.getByRole(AriaRole.BUTTON,
 ```
 
 ```csharp
-await Expect(page
+await Expect(Page
     .GetByRole(AriaRole.Heading, new() { Name = "Sign up" }))
     .ToBeVisibleAsync();
 
@@ -1641,7 +1641,7 @@ expect(page.get_by_title("Issues count")).to_have_text("25 issues")
 ```
 
 ```csharp
-await Expect(page.GetByTitle("Issues count")).toHaveText("25 issues");
+await Expect(Page.GetByTitle("Issues count")).toHaveText("25 issues");
 ```
 
 ## test-config-snapshot-path-template

--- a/docs/src/locators.md
+++ b/docs/src/locators.md
@@ -68,7 +68,7 @@ await page.GetByLabel("Password").FillAsync("secret-password");
 
 await page.GetByRole(AriaRole.Button, new() { Name = "Sign in" }).ClickAsync();
 
-await Expect(page.GetByText("Welcome, John!")).ToBeVisibleAsync();
+await Expect(Page.GetByText("Welcome, John!")).ToBeVisibleAsync();
 ```
 
 ## Locating elements
@@ -245,7 +245,7 @@ page.getByRole(AriaRole.BUTTON,
 ```
 
 ```csharp
-await Expect(page
+await Expect(Page
     .GetByRole(AriaRole.Heading, new() { Name = "Sign up" }))
     .ToBeVisibleAsync();
 
@@ -373,7 +373,7 @@ expect(page.get_by_text("Welcome, John")).to_be_visible()
 ```
 
 ```csharp
-await Expect(page.GetByText("Welcome, John")).ToBeVisibleAsync();
+await Expect(Page.GetByText("Welcome, John")).ToBeVisibleAsync();
 ```
 
 Set an exact match:
@@ -396,7 +396,7 @@ expect(page.get_by_text("Welcome, John", exact=True)).to_be_visible()
 ```
 
 ```csharp
-await Expect(page
+await Expect(Page
     .GetByText("Welcome, John", new() { Exact = true }))
     .ToBeVisibleAsync();
 ```
@@ -424,7 +424,7 @@ expect(page.get_by_text(re.compile("welcome, john", re.IGNORECASE))).to_be_visib
 ```
 
 ```csharp
-await Expect(page
+await Expect(Page
     .GetByText(new Regex("welcome, john", RegexOptions.IgnoreCase)))
     .ToBeVisibleAsync();
 ```
@@ -504,7 +504,7 @@ expect(page.get_by_title("Issues count")).to_have_text("25 issues")
 ```
 
 ```csharp
-await Expect(page.GetByTitle("Issues count")).toHaveText("25 issues");
+await Expect(Page.GetByTitle("Issues count")).toHaveText("25 issues");
 ```
 
 :::note When to use title locators
@@ -784,7 +784,7 @@ await expect(page.locator("x-details")).to_contain_text("Details")
 expect(page.locator("x-details")).to_contain_text("Details")
 ```
 ```csharp
-await Expect(page.Locator("x-details")).ToContainTextAsync("Details");
+await Expect(Page.Locator("x-details")).ToContainTextAsync("Details");
 ```
 ## Filtering Locators
 
@@ -910,7 +910,7 @@ expect(page.get_by_role("listitem").filter(has_not_text="Out of stock")).to_have
 
 ```csharp
 // 5 in-stock items
-await Expect(page.getByRole(AriaRole.Listitem).Filter(new() { HasNotText = "Out of stock" }))
+await Expect(Page.getByRole(AriaRole.Listitem).Filter(new() { HasNotText = "Out of stock" }))
     .ToHaveCountAsync(5);
 ```
 
@@ -1006,7 +1006,7 @@ expect(
 ```
 
 ```csharp
-await Expect(page
+await Expect(Page
     .GetByRole(AriaRole.Listitem)
     .Filter(new() {
         Has = page.GetByRole(AriaRole.Heading, new() { Name = "Product 2" })
@@ -1049,7 +1049,7 @@ expect(
 ```
 
 ```csharp
-await Expect(page
+await Expect(Page
     .GetByRole(AriaRole.Listitem)
     .Filter(new() {
         HasNot = page.GetByRole(AriaRole.Heading, new() { Name = "Product 2" })
@@ -1309,7 +1309,7 @@ assertThat(page.getByRole(AriaRole.LISTITEM).hasCount(3);
 ```
 
 ```csharp
-await Expect(page.GetByRole(AriaRole.Listitem)).ToHaveCountAsync(3);
+await Expect(Page.GetByRole(AriaRole.Listitem)).ToHaveCountAsync(3);
 ```
 
 ### Assert all text in a list
@@ -1349,7 +1349,7 @@ assertThat(page
 ```
 
 ```csharp
-await Expect(page
+await Expect(Page
     .GetByRole(AriaRole.Listitem))
     .ToHaveTextAsync(new string[] {"apple", "banana", "orange"});
 ```

--- a/docs/src/other-locators.md
+++ b/docs/src/other-locators.md
@@ -726,7 +726,7 @@ expect(page.locator("label")).to_have_text("Password")
 
 ```csharp
 // Fill the input by targeting the label.
-await Expect(page.Locator("label")).ToHaveTextAsync("Password");
+await Expect(Page.Locator("label")).ToHaveTextAsync("Password");
 ```
 
 ## Legacy text locator

--- a/docs/src/release-notes-csharp.md
+++ b/docs/src/release-notes-csharp.md
@@ -373,7 +373,7 @@ await page.GetByLabel("Password").FillAsync("secret-password");
 
 await page.GetByRole(AriaRole.Button, new() { NameString = "Sign in" }).ClickAsync();
 
-await Expect(page.GetByText("Welcome, John!")).ToBeVisibleAsync();
+await Expect(Page.GetByText("Welcome, John!")).ToBeVisibleAsync();
 ```
 
 All the same methods are also available on [Locator], [FrameLocator] and [Frame] classes.
@@ -387,7 +387,7 @@ All the same methods are also available on [Locator], [FrameLocator] and [Frame]
 - [`method: LocatorAssertions.toHaveAttribute`] with an empty value does not match missing attribute anymore. For example, the following snippet will succeed when `button` **does not** have a `disabled` attribute.
 
    ```csharp
-   await Expect(page.GetByRole(AriaRole.Button)).ToHaveAttribute("disabled", "");
+   await Expect(Page.GetByRole(AriaRole.Button)).ToHaveAttribute("disabled", "");
    ```
 
 ### Browser Versions

--- a/docs/src/writing-tests-csharp.md
+++ b/docs/src/writing-tests-csharp.md
@@ -50,7 +50,7 @@ public class Tests : PageTest
         await getStarted.ClickAsync();
 
         // Expects page to have a heading with the name of Installation.
-        await Expect(page
+        await Expect(Page
             .GetByRole(AriaRole.Heading, new() { Name = "Installation" }))
             .ToBeVisibleAsync();
     }

